### PR TITLE
Add section about switching login nodes

### LINKED ISSUE #551: Section about login nodes
For more information, see:
https://otrsdict.ugent.be/otrs/index.pl?Action=AgentTicketZoom&TicketID=126959

### PR DESCRIPTION
**Pull request by milachae**
closes #551

### CODE REVIEW

File: mkdocs/docs/HPC/connecting.md [hunk 1]
```diff
@@ -759,3 +759,17 @@ created.
 ### Fast file transfer for large datasets
 
 See [the section on `rsync` in chapter 5 of the Linux intro manual](../linux-tutorial/uploading_files/#copying-faster-with-rsync).
+
+## Changing login nodes
+It can be useful to work on the same login node using multiple terminal sessions. However, when you connect to the HPC (High-Performance Computing) system through SSH, you're directed to a random login node, which might not be the one where you already have an active session. To address this, there is a way to manually switch your active login node.
```
Comment (boegel) [inline mkdocs/docs/HPC/connecting.md:764]: I would rephrase this, it's mostly about having control over which login node you're on, not so much being able to have multiple sessions on the same login node.

The same applies when starting a shell session via the web portal, so I wouldn't mention SSH.

Also try to use "you are" rather than "you're".

File: mkdocs/docs/HPC/connecting.md [hunk 2]
```diff
@@ -759,3 +759,17 @@ created.
...
+
+For instance, if you want to switch to the login node named `gligar07`, you can use the following command while you are connected to the HPC:
+```shell
+ssh gligar07
```
Comment (boegel) [inline mkdocs/docs/HPC/connecting.md:768]: Mention that this makes sense when you're logged in to `gligar08`, and how to check where you're logged in (run the `hostname` command, show output of hostname before and after using `ssh`).

And also mention the reverse (use `ssh gligar08` if you got logged in to `gligar07` but want to be on `gligar08`).

File: mkdocs/docs/HPC/connecting.md [hunk 3]
```diff
@@ -759,3 +759,17 @@ created.
...
+```
+
+Rather than launching multiple terminals, you can also use a terminal multiplexer like `screen` or `tmux`.
```
Comment (boegel) [inline mkdocs/docs/HPC/connecting.md:771]: `screen` and `tmux` are mostly useful when you want to set up a shell session that "survives" across disconnects, not so much to launch multiple sessions

**Review by boegel — CHANGES_REQUESTED** [review]

File: mkdocs/docs/HPC/connecting.md (hunk 3 above)
Comment (milachae) [reply mkdocs/docs/HPC/connecting.md]: I changed this

File: mkdocs/docs/HPC/connecting.md [hunk 4]
```diff
@@ -759,3 +759,29 @@ created.
...
+For instance, if you want to switch to the login node named `{{loginhost}}`, you can use the following command while you are connected to the `{{altloginhost}}` login node on the HPC:
+```shell
+ssh {{loginhost}}
+```
+This is also possible the other way around.
+
+If you want to find out which login host you are connected to, you can use the `hostname` command.
+<pre><code>$ <b>hostname</b> 
+{{loginhost}}
+$ <b>ssh gligar08</b>
```
Comment (boegel) [inline mkdocs/docs/HPC/connecting.md:775]: Avoid hardcoding `gligar08`?

```suggestion
$ <b>ssh {{altloginhost}}</b>

**Review by boegel — CHANGES_REQUESTED** [review]

File: mkdocs/docs/HPC/connecting.md (hunk 4 above)
Comment (milachae) [reply mkdocs/docs/HPC/connecting.md:775]: Indeed, I forgot that one

**Review by boegel — APPROVED** [review]
